inception/inception/inception_distributed_train.py

```diff
@@ -110,7 +114,7 @@ def train(target, dataset, cluster_spec):
   is_chief = (FLAGS.task_id == 0)
 
   # Ops are assigned to worker by default.
-  with tf.device('/job:worker/task:%d' % FLAGS.task_id):
+  with tf.device('/job:worker/task:%d/gpu:%d' % (FLAGS.task_id, FLAGS.gpu_id)):
```

Comment:
What if the workers just use CPU and/or have not GPU?
Currently it would run in CPU if there is no GPU available in the worker.